### DISCUSSION TITLE
Duplicate content in feed

### DISCUSSION THREAD

**Discussion by kospl** [community/community#28481] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/26386660/183113738-ebfab16d-49c6-4a3e-ae4c-655a94b0968f.png)

**Comment by jfullstackdev** [comment]
seems this is one of those that needs to be fixed, like seeing repos that you already starred, hoping that one Staff will see this post

**Reply by kospl** [reply]
I thought I'm loosing my mind watching same record twice, isn't it weird?
